src/dxtbx/nexus/__init__.py

```diff
@@ -104,23 +105,18 @@ def get_dxtbx_scan(
     num_images = len(scan_axis)
     image_range = (1, num_images)
 
+    oscillation = (0, 0)
     if is_rotation:
         if scan_axis.end:
             oscillation = (
                 float(scan_axis[0].to("degree").magnitude),
                 float((scan_axis.end[0] - scan_axis[0]).to("degree").magnitude),
             )
-        # elif scan_axis.increment_set:
         elif num_images > 1:
             oscillation = (
                 float(scan_axis[0].to("degree").magnitude),
                 float((scan_axis[1] - scan_axis[0]).to("degree").magnitude),
             )
```

Comment:
```suggestion
            end = scan_axis[1]
        
        oscillation = (
            float(scan_axis[0].to("degree").magnitude),
            float((end - scan_axis[0]).to("degree").magnitude)
        )
```

Reply:
Looking at `dxtbx_model_ext.Scan.get_oscillation_range`, it seems this would still result in a valid single-image `Scan` object, plus it actually preserves the goniometer position in the `Scan`, rather than setting it to 0, thus preserving this piece of metadata in the DXTBX experiment model.  So it looks like my suggestion has no cost and might actually be a tiny enhancement (albeit one that probably no-one will ever notice).

Reply:
I think I've addressed this in https://github.com/cctbx/dxtbx/pull/455/commits/1dddae64727fa27e7d3b68c824c52ffbca0311f5 if I've understood your comments correctly